<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>749</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Franklin Gothic Demi Cond</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DETECTION APP</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_opcam">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IMAGE CAPTURED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(217, 217, 217);
	border-radius : 12px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed
{
	padding-left : 5px;
	padding-top : 5px;
	background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Choose Product 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid white ; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cb1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
    <item>
     <property name="text">
      <string>choose product</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>screwringmerah</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>screwringputih</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>screwringungu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>capungu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>caphijau</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>capmerahmuda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bottlemerah</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bottlebiru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bottleungu</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Choose Product 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Product Name Detected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pdnm">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Qty Product 1 Detected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="qty1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Qty Product 2 Detected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="qty2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Total Product 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tp_1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Total Product 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tp2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizebtn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi Cond</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: rgb(217, 217, 217);
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed
{
	padding-left : 5px;
	padding-top : 5px;
	background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closebtn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi Cond</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: rgb(217, 217, 217);
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed
{
	padding-left : 5px;
	padding-top : 5px;
	background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="datelbl">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>920</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DD : MM : YYYY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>890</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_det">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DETECT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cb2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
    <item>
     <property name="text">
      <string>choose product</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>screwringmerah</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>screwringputih</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>screwringungu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>capungu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>caphijau</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>capmerahmuda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bottlemerah</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bottlebiru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bottleungu</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-7</y>
      <width>971</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 68, 158);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startbtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(217, 217, 217);
	border-radius : 12px;
}

QPushButton:hover
{
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed
{
	padding-left : 5px;
	padding-top : 5px;
	background-color: rgb(217, 217, 217);
}
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="statusdet">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>580</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>lbl_opcam</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_3</zorder>
   <zorder>cb1</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>pdnm</zorder>
   <zorder>label_7</zorder>
   <zorder>qty1</zorder>
   <zorder>label_8</zorder>
   <zorder>qty2</zorder>
   <zorder>label_9</zorder>
   <zorder>tp_1</zorder>
   <zorder>label_10</zorder>
   <zorder>tp2</zorder>
   <zorder>minimizebtn</zorder>
   <zorder>closebtn</zorder>
   <zorder>datelbl</zorder>
   <zorder>timelabel</zorder>
   <zorder>lbl_det</zorder>
   <zorder>cb2</zorder>
   <zorder>startbtn</zorder>
   <zorder>statusdet</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
